packages/cashier/src/pages/payment-agent/payment-agent-list/payment-agent-list.tsx

```diff
@@ -26,6 +27,9 @@ const PaymentAgentList = ({ setSideNotes }: TProps) => {
                 <SideNote has_title={false} key={0}>
                     <PaymentAgentDisclaimer />
                 </SideNote>,
+                <SideNote has_title={false} key={0}>
```

Comment:
```suggestion
                <SideNote key={0}>
```
@Bahar-FS It will be `false` by default. Also I think `key` should be different 🤔